tests/Pagination.spec.js

```diff
@@ -23,9 +23,10 @@ describe('Uncontrolled Pagination', () => {
   function shouldHighlightRight() {
     const pagers = TestUtils.scryRenderedDOMComponentsWithTag(pagination, 'li');
     const current2 = pagination.state.current;
+    const totalText = 1;
```

Comment:
@benjycui i can rebase it remove the last commit that relate with the test case.
Then this pr will not pass The Travis CI.